tasks.rb

```diff
@@ -2439,6 +2439,7 @@ def create_sample_hpxmls
                   'base-hvac-central-ac-only-var-speed.xml',
                   'base-hvac-central-ac-plus-air-to-air-heat-pump-heating.xml',
                   'base-hvac-dse.xml',
+                  'base-hvac-ducts-area-fractions.xml',
```

Comment:
Run `openstudio tasks.rb update_measures` and add the sample file to the repo so we get CI results. You could also consider using that file in your test_hvac.rb unit test rather than taking a different sample file and modifying it to have duct fractions.